services/Cargo.toml

```diff
@@ -53,6 +54,7 @@ pwhash = "1.0"
 quick-xml = { version = "0.22", optional = true }
 rayon = "1.5"
 reqwest = { version = "0.11", features = ["json", "multipart", "stream"] }
+scienceobjectsdb_rust_api = { git = "https://github.com/ScienceObjectsDB/rust-api", branch="main" }
```

Comment:
Can't we move this dependency to only be used if nfdi is enabled?
It seems to be an unstable dependency by now.

Reply:
d760b3962b50d6acba8502aa4527c7b00ba83b5d